pkgs/by-name/tp/tpm2-pkcs11/package.nix

```diff
@@ -15,48 +16,59 @@
   python3,
   glibc,
   libyaml,
+  cmocka,
   abrmdSupport ? true,
-  tpm2-abrmd ? null,
   fapiSupport ? true,
+  fuzz ? false,
 }:
 
-stdenv.mkDerivation rec {
+clangStdenv.mkDerivation rec {
```

Comment:
Does the build only work with Clang?

Reply:
Nope, good call.

Reply:
The fuzz tests need it, but the fuzz tests currently fail. I just cased it out.